src/grid/style.scss

```diff
@@ -145,3 +151,81 @@
 		color: var(--stc-color-white);
 	}
 }
+
+.stc-grid-new-item{
+    font-family: 'Inter';
+    font-style: normal;
+}
+.stc-grid-new-item .stc-grid-item-header {
```

Comment:
Please write this in SCSS format, Like this , You can use this tool to convert to SCSS https://jsonformatter.org/css-to-scss
```
.stc-grid-new-item {
	font-family: 'Inter';
	font-style: normal;
	.stc-grid-item-header {
		padding: 20px 20px;
		border-top: 1px solid #e9e9e9;
		background: var(--stc-color-white);
		border: 1px solid var(--stc-border-color);
		box-sizing: border-box;
		border-radius: 0px 0px 4px 4px;
		display: grid;
	}
	.stc-grid-item-desc {
		font-family: 'Inter';
		font-style: normal;
		font-weight: 400;
		font-size: 15px;
		line-height: 25px;
	}
	.stc-grid-item-title {
		color: var(--st-color-heading);
		font-size: var(--st-font-size-xs);
		line-height: var(--st-line-height-xs);
		margin: 0;
		padding: 0;
		text-align: center;
		width: 100%;
		font-weight: 600;
		display: flex;
		align-items: center;
		line-height: 24px;
	}
	.stc-grid-item-hover-button-wrap {
		justify-content: center;
		position: absolute;
		left: 0;
		background: transparent;
		width: 100%;
		height: auto;
		visibility: hidden;
		opacity: 0;
		color: var(--stc-color-white);
		margin-top: 15px;
	}
}